server/webapp/WEB-INF/rails/webpack/views/components/site_menu/index.tsx

```diff
@@ -20,6 +20,10 @@ import styles from "./index.scss";
 
 const classnames = bind(styles);
 
+const activeItem = (href?: string): boolean => {
+  return href === window.location.pathname;
```

Comment:
Perhaps more robust would be a `startsWith()`, or even better, comparing `new URL(href).pathname === window.location.pathname`